spack_repo/fnal_art/packages/canvas_root_io/package.py

```diff
@@ -34,6 +34,9 @@ class CanvasRootIo(CMakePackage, FnalGithubPackage):
     version("1.09.04", sha256="cb854b4fdc72be24856886d985f96ceb3b0049729df0b4a11fb501ff7c48847b")
 
     patch("test_build.patch", when="@:1.11.00")
+    patch("https://github.com/art-framework-suite/canvas-root-io/commit/5d7c0bf55d54e907bd4be16d250da597aed595a5.patch",
+          sha256="bbb18b97a0d93bd3c8243172aab9ff4958fe9ab1d22e263c1fda19eebf827b47",
+          when="%root@6.36:")
```

Comment:
Can I suggest:

```suggestion
    patch("https://github.com/art-framework-suite/canvas-root-io/commit/5d7c0bf55d54e907bd4be16d250da597aed595a5.patch",
          sha256="bbb18b97a0d93bd3c8243172aab9ff4958fe9ab1d22e263c1fda19eebf827b47",
          when="@:1.14 %root@6.36:")
```
?
Otherwise, `develop` will fail to build.
Also, is there a reason you have `%root@6.36:` instead of `^root@6.36:`?

Reply:
`%` is direct dependence `^` is transitive dependence.